FRAuth/FRAuthTests/FRAuthSwiftTests/E2ETests/Callback-Live/AA-05-DeviceBindingCallbackTest.swift

```diff
@@ -39,6 +39,18 @@ class AA_05_DeviceBindingCallbackTest: CallbackBaseTest {
     }
     
     override func tearDown() {
+        let frUserKeys = FRUserKeys()
+        var userKeys: [UserKey] = []
+        userKeys = frUserKeys.loadAll()
+        
+        for (_, userKey) in userKeys.enumerated().reversed()
+        {
+            do {
+                try frUserKeys.delete(userKey: userKey, forceDelete: true)
+            }
+            catch {
+            }
```

Comment:
if we are not actually catching anything and don't care about the error, better to use `try? frUserKeys.delete(userKey: userKey, forceDelete: true)` without do/catch block

Reply:
I added a warning message in the catch that the deletion has failed...